Create property that determines if external IDs are shown inside location details modal (#377)
MapsPeople/web-ui: packages/map-template/.github/README.md

|`category`|`string`|If you want to indicate an active category on the map. The value should be the Key (Administrative ID). |
|`searchAllVenues`|`bool`|If you want to perform search across all venues in the solution. |
|`hideNonMatches`|`bool`|Determine whether the locations on the map should be filtered (only show the matched locations and hide the rest) or highlighted (show all locations and highlight the matched ones with a red dot by default). If set to true, the locations will be filtered. |

> |`showExternalIDs`|`bool`|Determine whether the location details on the map should have an external ID visible. The default value is set to false. |

Reviewer: ```suggestion
|`showExternalIDs`|`bool`|Determine whether the Location details should show the external ID or not. The default value is set to false. |
```

|`searchAllVenues`|`bool`|If you want to perform search across all venues in the solution. |
|`hideNonMatches`|`bool`|Determine whether the locations on the map should be filtered (only show the matched locations and hide the rest) or highlighted (show all locations and highlight the matched ones with a red dot by default). If set to true, the locations will be filtered. |
|`show-road-names`|`showRoadNames`|`bool`|A boolean parameter that dictates whether Mapbox road names should be shown. By default, Mapbox road names are hidden when MapsIndoors data is shown. It is dictated by `mi-transition-level` which default value is 17. |

> |`showExternalIDs`|`bool`|Determine whether the Location details should show the external ID or not. The default value is set to false. |

Reviewer 2: ```suggestion
|`show-external-ids`|`showExternalIDs`|`bool`|Determine whether the Location details should show the external ID or not. The default value is set to false. |
```
